### DISCUSSION TITLE
Largo patches framing are not identical to the corresponding framing in the video

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#943] [Q&A]
I noticed a strange thing in video labelling that I replicated with the Biigle Demo video.
1. I jump to some arbitrary point in the video: in this case time 00:01:51:21.
2. I draw a circle with a label, say biota.
3. I now go to largo and query all biota cut outs.
4. I see that the octopus that was enclosed inside the circle is not off to its side. 
See following images:
In the video page: 
![image](https://github.com/user-attachments/assets/99a6a595-85ee-4b16-a99c-ce649485c358)
In the largo page:
![image](https://github.com/user-attachments/assets/4c1b2838-ef63-4109-b8fb-4e630049ced2)

 observed the following. When I use the "experimental" frame to frame progression in the video, I see that 01:51:21 does not seem to contain a valid frame (because I just randomly clicked on the video timeline to this location and it showed me a frame where I did my annotation). 
The frame progression goes when I jump frame my frame,  01:51:20, 01:51:24. Does that in your opinion cause this issue?

The impact of this offset is that I cannot effectively use Largo to perform relabeling since it does not always show the whole underlying fauna meaning that I have to always check the video to confirm.

**Comment by mzur** [comment]
Thanks for reporting this issue! Actually, the Largo thumbnail is correct and the video is not displayed correctly in the annotation tool. The root issue is described here: https://github.com/biigle/core/issues/391

Basically, browsers display the videos correctly only if the "moov atom" of the MP4 file is placed properly. As it happens, this wasn't the case for our demo video. I have now uploaded a fixed version of the demo video. Now you will see that your annotation suddenly matches with the Largo thumbnail. If you keep an eye on these moov atoms in your own videos and fix their position if necessary (see [here](https://github.com/biigle/core/issues/391#issuecomment-1470201751)), everything should work as expected.
